package.json

```diff
@@ -37,15 +37,20 @@
         "eslint": "8.43.0",
         "eslint-config-next": "13.4.7",
         "mongoose": "^7.4.2",
-        "next": "13.4.7",
-        "next-auth": "^4.24.7",
-        "postcss": "8.4.24",
+        "next": "^14.2.15",
+        "next-auth": "^4.24.8",
+        "postcss": "^8.4.47",
+<<<<<<< Updated upstream
```

Comment:
You should clean this area up. Otherwise, it might confuse other developers.

Reply:
Hello guys, thank you for the feedback and sorry for the late response. I have officially completed this and it should work. Let me know on your side.